plugins/action_generate_document/src/generateDoc.ts

```diff
@@ -18,6 +18,9 @@ const generateDoc: ActionPluginType = async ({
     const result = await generatePDF({
       fileId: docTemplateId,
       data,
+      options: {
+        lang: 'pt-pt',
+      },
```

Comment:
Doesn't this need to be a parameter, not hard-coded?

Reply:
Yes - good point. Done in a hurry. I'll fix today